portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,51 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kathleen's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 class="rainbow">Kathleen's Portfolio</h1>
+      <h4 style="background-color:Lavender">Hello! 
+        I am Kathleen Qian~ Welcome to my portfolio.</h4>
```

Comment:
Was the ~ intended?

Reply:
Yes... Don't want to be that formal... While I can change it if needed.

Reply:
I have adjusted that to reduce confusion.